script/index.js

```diff
@@ -0,0 +1,43 @@
+const speaker = [
+  {
+    simage: './images/pic.png',
+    sName: 'Ntare G',
+    sLocation: 'Rwanda, Kigali',
+    sBio: 'A associate Software developer at microverse with  expertise in Full-stack development and DevOps.',
+  },
```

Comment:
- [ ] [OPTIONAL] it doesn't seem necessary to prefix the object's keys with an `s`. we already know that each of the objects is a single speaker. so you could just remove the prefix from these keys.